### PR TITLE
[EP-802] HomeScreen refactor + UI changes

### PR DESCRIPTION
**Pull request by AndreiGalkin**
https://thedojo.atlassian.net/browse/EP-802

Kitten

### CODE REVIEW

File: sample-app/android/app/release/output.json [hunk 1]
```diff
@@ -0,0 +1 @@
+[{"outputType":{"type":"APK"},"apkData":{"type":"MAIN","splits":[],"versionCode":1,"versionName":"1.0","enabled":true,"outputFile":"app-release.apk","fullName":"release","baseName":"release","dirName":""},"path":"app-release.apk","properties":{}}]
```
Comment (stpopa) [inline sample-app/android/app/release/output.json:1]: please remove this file

File: sample-app/screens/DataConfig.tsx [hunk 2]
```diff
@@ -229,3 +230,88 @@ export var SettingsData = {
      }
   ]
 }
+
+export enum HomeListType {
```
Comment (stpopa) [inline sample-app/screens/DataConfig.tsx:234]: for consistency, can we keep the same naming a in Android SDK:
```
export enum DemoFeature {
  Payment,
  PreAuth,
  CreateCardToken,
  CheckCard,
  SaveCard,
  GooglePayPayment,
  GooglePayPreAuth,
  ApplePayPayment,
  ApplePayPreAuth,
  PaymentMethods,
  PreAuthPaymentMethods
}
```
there's no `ideal` option for now
and let's use `PascaCase` for enums https://www.typescriptlang.org/docs/handbook/enums.html

File: sample-app/android/app/release/output.json (hunk 1 above)
Comment (AndreiGalkin) [reply sample-app/android/app/release/output.json:1]: 👍

File: sample-app/screens/DataConfig.tsx (hunk 2 above)
Comment (AndreiGalkin) [reply sample-app/screens/DataConfig.tsx:234]: ok, I will change all enums in the project

**Pull request merged**